packages/frontend/src/features/sqlRunner/components/visualizations/BarChart.tsx

```diff
@@ -0,0 +1,34 @@
+import EChartsReact from 'echarts-for-react';
+import { type EChartsReactProps } from 'echarts-for-react/lib/types';
+import { memo, type FC } from 'react';
+import { useSelector } from 'react-redux';
+import { type useSqlQueryRun } from '../../hooks/useSqlQueryRun';
+import { useBarChartDataTransformer } from '../../transformers/useBarChartDataTransformer';
+
+type BarChartProps = Omit<EChartsReactProps, 'option'> & {
+    data: NonNullable<ReturnType<typeof useSqlQueryRun>['data']>;
+};
+
+const BarChart: FC<BarChartProps> = memo(({ data, className, ...rest }) => {
+    // TODO: fix store type
+    const barChartConfig = useSelector(
```

Comment:
```suggestion
    const barChartConfig = useAppSelector(
```

we have 2 hooks that should be used instead of importing the ones from `react-redux` - types are inferred correctly

Reply:
👍 Cool, I didn't know that.